src/app/home-page/Progress.ts

```diff
@@ -0,0 +1,5 @@
+export interface Progress {
+  current: number;
+  total: number;
+  bar: string;
+}
```

Comment:
this is the new file to export progress interface. But it's put with the home-page folder, idk if the place/order would make any difference or not.